metafacture-runner/src/main/dist/examples/gnd/references/merge_fix.flux

```diff
@@ -0,0 +1,23 @@
+
+default base = "";
+default counts= base + "counts.dat";
+default gndsimple = base + "gnd-simplified.dat";
+default out = base + "gnd-references.csv";
+
+//merge and output
+"megring information" | write("stdout");
+
+counts + "," + gndsimple
+| decode-string(",")
+| open-file
+| as-lines
+| decode-formeta
+| batch-log("records read: ${totalRecords}",batchsize="100000")
+| stream-to-triples
+| decouple
+| sort-triples(by="subject")
+| collect-triples
+| fix(FLUX_DIR + "output.fix")
+| batch-log(batchsize="100000")
+| encode-csv(separator=";",noQuotes="true")
```

Comment:
The Morph example produces triples instead of CSV (`stream-to-triples` + `template`).

Reply:
While the workflow with the morph streams triples it the workflow still outputs CSV at the end. I think that creating a csv with morph was way more difficult due to the streaming mechanism and no possibility to set an order of the output.
Instead of the hastle with the workaround morph->triple->template with fix I encode to csv easily.

Reply:
Also see: https://github.com/metafacture/metafacture-core/blob/a5f6eec225d3d0da0aaf8723144c98679e9cb1aa/metafacture-runner/src/main/dist/examples/gnd/references/output.xml#L5-L7